packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -23,7 +23,7 @@ export const validAddress = (value: string) => !/[`~!$%^&*_=+[}{\]\\"?><|]+/.tes
 export const validPostCode = (value: string) => value === '' || /^[A-Za-z0-9][A-Za-z0-9\s-]*$/.test(value);
 export const validTaxID = (value: string) => /(?!^$|\s+)[A-Za-z0-9./\s-]$/.test(value);
 export const validPhone = (value: string) => /^\+?([0-9-]+\s)*[0-9-]+$/.test(value);
-export const validLetterSymbol = (value: string) => /^[A-Za-z]+([a-zA-Z.' -])*[a-zA-Z.' -]+$/.test(value);
+export const validLetterSymbol = (value: string) => /^[A-Za-z][A-Za-z.' -]*[A-Za-z.' -]$/.test(value);
```

Comment:
Please write unit test cases

Reply:
@balakrishna-deriv I have write the test cases can you please check and approve it